_posts/2017-05-17-forbes.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Forbes
+sourceURL: https://www.youtube.com/watch?v=JmC0xkCMFCE
+image: /images/forbes.png
+tags:
+- News
+- Notifications
+- Service Worker
+- Offline
+
+---
+
+Forbes redesigned their mobile experience as a PWA resulting in **43% increase** in sessions per user, ad viewability up **20%**, and **100% more engagement**.
```

Comment:
> Are these "rules" on what should be bolded or not bolded written down somewhere?

Nope! Might be a good idea, though. Once we decide on rules we _like_, anyway.

> - I find the numbers bolded on their own doesn't add meaning. 15% What is the significance of the 15%?
> - I've encountered some items that don't have a number. For example, "look at more products."

This is fair feedback. Maybe we could simplify to bolding one highlight per sentence or something.

Since it looks like there's already inconsistency in bolding, I'm open to moving forward as-is and creating an issue or something to standardize and make note of the rules.

> Updating the site is already too time-consuming. 

You're actually going above and beyond what we expect of contributors! If you click the "Contribute a Story" button you can see it just creates a new issue so us dev folk can quickly whip things together without making individual contributors jump through these hoops. But you're more than welcome to submit PRs, too!